packages/account/src/Sections/Security/TwoFactorAuthentication/digit-form.tsx

```diff
@@ -1,64 +1,74 @@
 import React from 'react';
 import classNames from 'classnames';
-import { Formik, Form, Field } from 'formik';
+import { Formik, Form, Field, FieldProps, FormikProps } from 'formik';
 import { Input, Button } from '@deriv/components';
-import { getPropertyValue, WS } from '@deriv/shared';
+import { useSendUserOTP } from '@deriv/api';
 import { localize } from '@deriv/translations';
+import { useStore } from '@deriv/stores';
 
-const DigitForm = ({ is_enabled, setTwoFAStatus, setTwoFAChangedStatus, is_language_changing }) => {
-    const [is_success, setSuccess] = React.useState(false);
-    const [is_ready_for_verification, setReadyForVerification] = React.useState(false);
-    const button_text = is_enabled ? localize('Disable') : localize('Enable');
-    const formik_ref = React.useRef();
-    let enable_response;
+type TDigitFormValues = {
+    digit_code: string;
+};
+
+const DigitForm = () => {
```

Comment:
do we need a store observer here?
`const DigitForm = observer(() => { ...`

Reply:
works fine without it as well. Shall I add it?